examples/multipart/index.js

```diff
@@ -2,42 +2,42 @@
  * Module dependencies.
  */
 
-var express = require('../..');
-var multiparty = require('multiparty');
-var format = require('util').format;
+const express = require('../..');
+const multiparty = require('multiparty');
+const format = require('util').format;
 
-var app = module.exports = express();
+const app = module.exports = express();
 
-app.get('/', function(req, res){
+app.get('/',(req, res) => {
   res.send('<form method="post" enctype="multipart/form-data">'
     + '<p>Title: <input type="text" name="title" /></p>'
     + '<p>Image: <input type="file" name="image" /></p>'
     + '<p><input type="submit" value="Upload" /></p>'
     + '</form>');
 });
 
-app.post('/', function(req, res, next){
+app.post('/', (req, res, next) => {
```

Comment:
inconsistency: space before callback, other places don't have spaces

Reply:
Which space do you mean? To me this looks right, and seems to be following [standard style](https://standardjs.com)

Reply:
This is the correct formatting AFAIK for standard.  Any places without the space should be updated to follow this example.  Like line 11 of this file, it should have a space after the comma.